users/migrations/0005_auto_20190629_0830.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.2 on 2019-06-29 08:30
+
+from django.db import migrations, models
+import users.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0004_auto_20190627_0630'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='customuser',
+            name='email',
+            field=models.EmailField(error_messages={'blank': 'Enter your email address, please.'}, max_length=255, unique=True, validators=[users.validators.UserEmailValidation()], verbose_name='email address'),
+        ),
+        migrations.AlterField(
+            model_name='customuser',
+            name='password',
+            field=models.CharField(error_messages={'null': 'Enter you password, please.'}, max_length=128),
+        ),
+    ]
```

Comment:
Please delete this two migrations and generate one with all operations. Or maybe what you doing requires two migrations. If yes please explain why :)

Reply:
Done